cargo-cyclonedx/src/author.rs

```diff
@@ -36,11 +33,36 @@ pub enum AuthorParseError {
     Email(String),
 }
 
-#[derive(Debug, Serialize, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Debug)]
+pub struct Authors(pub Option<Vec<Author>>);
```

Comment:
We'll need a follow-up issue to track authors:

- Drop the email regex and replace it with a dependency?
- Better handling of invalid authors